val list = listOf(1, 2, 3)
println(list[3])







/*
fun main() {

    val list = listOf(1, 2, 3)

    try {
        println(list[3]) // This will throw an IndexOutOfBoundsException
    } catch (e: IndexOutOfBoundsException) {
        //println("Caught an IndexOutOfBoundsException: ${e.message}")
    }
}
*/